File: 프로그래머스/lv3/12987. 숫자 게임/숫자 게임.py
def solution(A, B):
    A.sort()
    B.sort()
    n = len(A)
    cnt = 0
    idx = n - 1
    for i in range(n - 1, -1, -1):
        if A[i] < B[idx]:
            idx -= 1
            cnt += 1
    return cnt

File: 프로그래머스/lv3/12987. 숫자 게임/숫자 게임.java
import java.util.*;


class Solution {
    public int solution(int[] A, int[] B) {
        Arrays.sort(A);
        Arrays.sort(B);
        int idx = A.length - 1;
        int cnt = 0;
        
        for (int i = A.length - 1; i > -1; i--){
            if (A[i] < B[idx]){
                cnt++;
                idx--;
            }
        }
        
        return cnt;
    }
}